components/validator/flags/flags.go

```diff
@@ -3,120 +3,123 @@ package flags
 import (
 	"time"
 
-	"github.com/urfave/cli"
+	"github.com/urfave/cli/v2"
 
-	kservice "github.com/kroma-network/kroma/utils/service"
 	klog "github.com/kroma-network/kroma/utils/service/log"
 	kmetrics "github.com/kroma-network/kroma/utils/service/metrics"
 	kpprof "github.com/kroma-network/kroma/utils/service/pprof"
 	krpc "github.com/kroma-network/kroma/utils/service/rpc"
 	"github.com/kroma-network/kroma/utils/service/txmgr"
 )
 
-const envVarPrefix = "VALIDATOR"
+const EnvVarPrefix = "VALIDATOR"
+
+func prefixEnvVars(name string) []string {
+	return []string{EnvVarPrefix + "_" + name}
+}
```

Comment:
How about using utils function? Please apply same changes for the functions in node, batcher, txmgr flag!
```suggestion
func prefixEnvVar(name string) []string {
	return kservice.PrefixEnvVar(EnvVarPrefix, name)
}
```